### ISSUE TITLE
Multipolygon: Alerts

### ISSUE THREAD

**Issue by WxBDM** [WxBDM/nwsapy#7]
When attempting to do benchmark tests, I noticed that there's a MultiPolygon when fetching all alerts. APy doesn't handle multipolygons as of 0.2.1. The logic to handle multipolygons would be in the ~`_construct_geometry()`~ EDIT: `_format_geometry()` method in `Individual Alerts`. May also want to consider bringing back the multipolygon with Shapely and include it as an attribute.

Details:
```
Traceback (most recent call last):
  File "/Users/bdmolyne/PycharmProjects/test/nwsapy_test.py", line 13, in <module>
    alerts = nwsapy.get_all_alerts(area = "FL")
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/nwsapy/nwsapy.py", line 140, in get_all_alerts
    return alerts.AllAlerts(self._user_agent_to_d, param_d)
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/nwsapy/alerts.py", line 458, in __init__
    self._init_active_and_all(response)
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/nwsapy/alerts.py", line 381, in _init_active_and_all
    self.alerts = [IndividualAlert(x) for x in info if x['properties']['event'] != 'Test Message']
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/nwsapy/alerts.py", line 381, in <listcomp>
    self.alerts = [IndividualAlert(x) for x in info if x['properties']['event'] != 'Test Message']
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/nwsapy/alerts.py", line 151, in __init__
    geom_d = self._format_geometry(alert_list['geometry'])
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/nwsapy/alerts.py", line 173, in _format_geometry
    points = [Point(x[0], x[1]) for x in geometries['coordinates'][0]]
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/nwsapy/alerts.py", line 173, in <listcomp>
    points = [Point(x[0], x[1]) for x in geometries['coordinates'][0]]
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/shapely/geometry/point.py", line 48, in __init__
    self._set_coords(*args)
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/shapely/geometry/point.py", line 137, in _set_coords
    self._geom, self._ndim = geos_point_from_py(tuple(args))
  File "/Users/bdmolyne/PycharmProjects/test/venv/lib/python3.9/site-packages/shapely/geometry/point.py", line 214, in geos_point_from_py
    dx = c_double(coords[0])
TypeError: must be real number, not list
```

<img width="870" alt="Screenshot of information" src="https://user-images.githubusercontent.com/25293211/122685556-b03ba100-d1d1-11eb-9e32-7d0adc665cc4.png">

Link: https://api.weather.gov/alerts/urn:oid:2.49.0.1.840.0.9cd140618699a0fa7c1e8a11ca93369eb8a207e2.006.1

Multipolygon structure (for documentation purposes):
```python
[[[[-86.92, 30.371], [-87.09700000000001, 30.334], [-87.21599990000001, 30.32], [-87.28400000000002, 30.31], [-87.30200000000002, 30.328999999999997], [-87.28900000000002, 30.336999999999996], [-87.18500000000002, 30.343999999999998], [-87.18200000000002, 30.342999999999996], [-87.07800000000002, 30.350999999999996], [-87.05200000000002, 30.358999999999995], [-86.92800000000003, 30.380999999999993], [-86.92, 30.371]]], [[[-87.16100000000003, 30.514999999999993], [-87.15900000000003, 30.496999999999993], [-87.15000000000003, 30.48999999999999], [-87.16200000000003, 30.478999999999992], [-87.16000000000004, 30.467999999999993], [-87.16300000000004, 30.426999999999992], [-87.19199990000004, 30.41399999999999], [-87.20800000000004, 30.40099999999999], [-87.22500000000004, 30.39899999999999], [-87.23900000000003, 30.39599999999999], [-87.26700000000004, 30.37399999999999], [-87.26700000000004, 30.34399999999999], [-87.30600000000004, 30.34599999999999], [-87.31500000000004, 30.335999999999988], [-87.31399990000004, 30.326999999999988], [-87.32100000000004, 30.316999999999986], [-87.38000000000004, 30.305999999999987], [-87.55300000000004, 30.270999999999987], [-87.55300000000004, 30.274999999999988], [-87.55400000000004, 30.282999999999987], [-87.55100000000004, 30.28699999999999], [-87.54200000000004, 30.288999999999987], [-87.51500000000004, 30.300999999999988], [-87.49900000000004, 30.317999999999987], [-87.49000000000004, 30.319999999999986], [-87.44199990000004, 30.323999999999987], [-87.43400000000004, 30.338999999999988], [-87.41500000000003, 30.346999999999987], [-87.42200000000004, 30.358999999999988], [-87.42800000000004, 30.352999999999987], [-87.44100000000005, 30.36599999999999], [-87.42500000000004, 30.37499999999999], [-87.43100000000004, 30.38999999999999], [-87.42600000000004, 30.40199999999999], [-87.39800000000004, 30.40099999999999], [-87.39100000000003, 30.41099999999999], [-87.35800000000003, 30.41699999999999], [-87.34000000000003, 30.43099999999999], [-87.35500000000003, 30.44499999999999], [-87.36500000000004, 30.45799999999999], [-87.38900000000004, 30.44599999999999], [-87.41700000000004, 30.452999999999992], [-87.43800000000005, 30.48099999999999], [-87.43600000000005, 30.48899999999999], [-87.42900000000004, 30.48799999999999], [-87.43300000000005, 30.49799999999999], [-87.43800000000005, 30.49399999999999], [-87.43800000000005, 30.50399999999999], [-87.44199990000004, 30.50299999999999], [-87.44799990000004, 30.51299999999999], [-87.44600000000004, 30.52199999999999], [-87.37400000000004, 30.56199999999999], [-87.29300000000003, 30.57699999999999], [-87.26500000000003, 30.61199999999999], [-87.25700000000003, 30.61499999999999], [-87.25900000000003, 30.61199999999999], [-87.24200000000003, 30.60599999999999], [-87.24600000000004, 30.59899999999999], [-87.23300000000003, 30.594999999999988], [-87.22500000000004, 30.581999999999987], [-87.21900000000004, 30.568999999999985], [-87.20800000000004, 30.554999999999986], [-87.20099990000004, 30.553999999999984], [-87.19499990000004, 30.549999999999983], [-87.18000000000004, 30.543999999999983], [-87.18700000000004, 30.540999999999983], [-87.18800000000005, 30.531999999999982], [-87.16100000000003, 30.514999999999993]]]]
```

**Comment by WxBDM** [comment]
Good news: I fixed it and APy handles multipolygons properly. Instead of creating a Shapely MultiPolygon, it'll construct a list of polygons, which the end user can instantiate a Shapely MultiPolygon list. The reasoning for this is the same as to why there isn't a MultiPoint attribute: it's not necessary.

Bad news: constructing individual alert series logic is broken. Early on, I was thinking of `to_csv()` to be the primary usage of the series (not really sure why), and all the data types are messed up as a result. I think a re-approach to `to_dataframe()` is needed: don't reformat, just build the series directly from the attribute dictionary during object construction.

I'm leaving this issue open until the above "bad news" gets resolved.

Edit: This will be implemented in v0.2.2

**Comment by WxBDM** [comment]
Closed due to rewrite of v1.0.0
